src/pages/comparison/comparison.tsx

```diff
@@ -460,6 +502,99 @@ export const Comparison = ({ mode }: ComparisonPageProps) => {
     return result;
   };
 
+  const handleInsertExample = (
+    newLayer: LayerExample,
+    side: "left" | "right"
+  ) => {
+    switch (side) {
+      case "left":
+        setLeftSideExamples((prevValues) => [...prevValues, newLayer]);
+        setLayerLeftSide(newLayer);
+        setLeftSideSelectedExampleId(newLayer.id);
+        break;
+      case "right":
+        setRightSideExamples((prevValues) => [...prevValues, newLayer]);
```

Comment:
We need to check that the `id` is unique. Can be done on separate PR. We should create a Jira ticket for it.

Reply:
Yes Sure.